spec/services/claim_reporter_spec.rb

```diff
@@ -25,7 +25,7 @@
 
   describe '#completion_rate' do
     before do
-      Timecop.freeze(5.weeks.ago) do
+      travel -5.weeks do
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3627/files#r538196976)